packages/remix-dev/cli/commands.ts

```diff
@@ -234,3 +235,10 @@ export async function codemod(
     throw error;
   }
 }
+
+export async function dev2(projectDir?: string) {
+  projectDir ??= process.cwd();
+  let config = await readConfig(projectDir);
+  await devServer2.serve(config);
+  await new Promise(() => {});
+}
```

Comment:
alternatively, we could detect `v2_dev` in future flags and change the behavior of `remix dev`.